src/main/java/org/opensearch/security/dlic/rest/validation/SavedTenantValidator.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Portions Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The license header is old. Could you please use [this one](https://github.com/sandeshkr419/security/blob/main/src/test/java/org/opensearch/security/DataStreamIntegrationTests.java#L1-L14).

Reply:
Changed